# This converts valid evm addresses to the checksummed version (`evm-address-to-checksummed/evm-address-to-checksummed`)

ðŸ”§ This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->

Please describe the origin of the rule here.

## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js

let unCheckSummedAddress1 = '0xaaa6bf7f35f1cb1afb9f5375e363140c9b197274';
let unCheckSummedAddress2 = '0x4c906d3ecfae8a4cc21edf7e7a17dbd5b38d41fd';

```

Examples of **correct** code for this rule:

```js

// fill me in
let correctlyCheckSummedAddress1 = "0xAaa6Bf7F35f1CB1AFb9f5375E363140C9B197274"
let corrctlyCheckSummedAddress2 = "0x4c906D3EcFaE8A4cC21Edf7E7A17dBd5B38D41fd"

```

### Options


## When Not To Use It

When you don't want checksummed addresses

## Further Reading

